utils/get-project-content.js

```diff
@@ -121,14 +128,9 @@ function _getCollectionFragments(collectionDirectory) {
           )
         };
 
-        if (
-          metadata.thumbnailPath &&
-          fs.existsSync(path.resolve(directory, metadata.thumbnailPath))
-        ) {
+        if (metadata.thumbnailPath && fs.existsSync(metadata.thumbnailPath)) {
           return Object.assign(fragment, {
-            thumbnail: fs.createReadStream(
-              path.resolve(directory, metadata.thumbnailPath)
-            )
+            thumbnail: fs.createReadStream(metadata.thumbnailPath)
```

Comment:
Storing the full ReadStream object inside projectContent case problems in the compress method, we should build a more JSON-friendly object here, or use the thumbnailPath when the image is needed

Reply:
fixed